### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by aamir74**
Navbar component added.
navbar

### CODE REVIEW

**Review by Sudhanshusoni45 — APPROVED** [review]
Code is neat and clean and well written, great work.

File: index.html [hunk 1]
```diff
@@ -599,6 +604,38 @@ <h2>Round Images</h2>
...
+            </div>
+          </nav>
+          <div class="p-code">
+            <iframe src="https://carbon.now.sh/embed/WLGVrI0mytkOxAP20MOC" style="
+                  width: 680px;
+                  height: 400px;
+                  border: 0;
+                  transform: scale(1);
+                  overflow: hidden;
+                " sandbox="allow-scripts allow-same-origin">
```
Comment (Jay-Gadhiya) [inline index.html:634]: you can make one class for it and put that style there.

File: styles.css [hunk 2]
```diff
@@ -53,8 +53,8 @@
   width: 100%;
   padding: 1rem;
   z-index: 2;
-  color:var(--white);
-  display:flex;
+  color: var(--white);
```
Comment (Jay-Gadhiya) [inline styles.css:56]: just a suggestion, you can give variable name as "--white-color" for better understanding

File: styles.css [hunk 3]
```diff
@@ -118,6 +118,4 @@
   article {
     width: 300px;
   }
```
Comment (Jay-Gadhiya) [inline styles.css:120]: suggestion: ignore using direct tag names, just put class and give style to the class.

File: index.html [hunk 4]
```diff
@@ -76,6 +76,11 @@ <h2>Plus UI</h2>
...
         </div>
       </a>
+      <a class="nav__item" href="#navbar">
+        <div>
+          <i class="fa fa fa-bars nav-icon" aria-hidden="true"></i><span class="nav-name">Navbar</span>
+        </div>
+      </a>
     </a>
   </aside>
   <content class="p-content">
```
Comment (prajapati-priyanka) [inline index.html:86]: Seen <**content**> tag first time.  Will definitely read about this tag.

**Review by vermasavita — APPROVED** [review]
Small suggestion: change the background color of badge icon. It's hard to see when colors are the same. :-)

**Review by Jay-Gadhiya — APPROVED** [review]
- your navbar looks good
  - just one suggestion,  in the navbar icon badge (cart icon badge) you can give a border to more clearly show because its color is overlap with the navbar background color.
  - code looks clean

**Review by prajapati-priyanka — APPROVED** [review]
Good use of semantic HTML.  UI is good.  Great Work.

**Pull request merged**